src/common/components/Textarea/Textarea.theme.ts

```diff
@@ -19,6 +20,10 @@ export const textareaTheme = tv(
         "bg-surface-base",
         "border-border-default",
         "text-text-em-mid",
+        "hover:border-surface-elevated",
+        "hover:bg-surface-elevated",
+        "focus-visible:bg-bg-base",
+        "focus-visible:outline-none",
```

Comment:
question: should these be added to [`@/common/styles/components.scss#L3`](https://github.com/AfterClass-io/afterclass.io-v2/blob/main/src/common/styles/components.scss#L3) instead? to standardize the focus ring behavior across all components that utilize it

cc @Aztriltus

Reply:
Yeah would appreciate some input on this the reason I did not want to add those behaviors in is from my understanding focus ring is also used in other components such as the button

Reply:
Ammended!

Reply:
@Aztriltus would it be better to leave out the "focus-visible:bg-bg-base" out from focus ring as it would also affect the current button's behavior which I do not think is intended 
<img width="1043" alt="Screenshot 2024-05-08 at 2 48 24 PM" src="https://github.com/AfterClass-io/afterclass.io-v2/assets/72653120/8d667218-d51a-40db-a194-574eea187947">